src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -9563,7 +9563,7 @@ void Unit::UpdateCharmAI()
         UnitAI* newAI = nullptr;
         if (GetTypeId() == TYPEID_PLAYER)
         {
-            if (Unit* charmer = GetCharmer())
+            if (Unit* charmer = GetCharmer() || GetCharmer() == TYPEID_PLAYER)
```

Comment:
i suppose not intended change

Reply:
Anyway, that change is the reason why the build fails.

Reply:
Non intented change, will revert.